src/Test/ResetDatabase.php

```diff
@@ -50,11 +50,6 @@ public static function _resetDatabase(): void
      */
     public static function _resetSchema(): void
     {
-        if (DatabaseResetter::isDAMADoctrineTestBundleEnabled()) {
-            // not required as the DAMADoctrineTestBundle wraps each test in a transaction
-            return;
-        }
```

Comment:
Now even with dama, the kernel will be booted: we need to know whether or not we have odm in the kernel, this condition has been moved into `ORMDatabaseResetter::resetSchame()`